JobRunner/JobRunner.py

```diff
@@ -36,7 +37,7 @@ class JobRunner(object):
     to support subjobs and provenenace calls.
     """
 
-    def __init__(self, config: Config, port=None):
+    def __init__(self, config: Config, port=None, server_mode: bool = False):
```

Comment:
It would be good to add info about what "Server Mode" means in the comment

Reply:
Fixed